Add env variables for the agent (#244)
seqeralabs/docs: platform_versioned_docs/version-24.1/compute-envs/hpc.mdx

You can also use workspace [SSH credentials](../credentials/ssh_credentials.mdx) for cluster login, but this provides service account access to your HPC to all Platform users. This means that all users will be granted the same file system access, and all activity is logged under the same user account on your HPC cluster. 

For HPC clusters that do not allow direct access through an SSH client, a secure connection can be authenticated with [Tower Agent](../supported_software/agent/agent.mdx).

## Work and launch directories

> For instances where the work directory or launch directory must be set dynamically at runtime you can use variable expansion. This works in conjunction with Tower Agent. The path that results from variable expansion must exist prior to workflow execution as the agent does not create missing directories.

Reviewer: ```suggestion
For instances where the work directory or launch directory must be set dynamically at runtime, you can use variable expansion. This works in conjunction with Tower Agent. The path that results from variable expansion must exist before workflow execution as the agent does not create directories.
```

The following variables are supported:

- `TW_AGENT_WORKDIR`: Resolves to the work directory for Tower Agent. By default, this directory resolves to the `${HOME}/work` path, where `HOME` is the home directory of the user that the agent runs as. The work directory can be overridden by specifying the `--work-dir` argument when configuring Tower Agent. For more information, see the [Tower Agent][agent] documentation.
- `TW_AGENT_USER`: Resolves to the username that the agent is running as. By default, this is the Unix username that the agent runs as. On systems where the agent cannot determine which user it runs as, it falls back to the value of the `USER` environment variable.

## HPC compute environment

To create a new **HPC** compute environment:

1.  In a Seqera workspace, select **Compute environments > New environment**.
1.  Enter a descriptive name for this environment. Use only alphanumeric characters, dashes, and underscores.
1.  Select your HPC environment from the **Platform** dropdown menu.
1.  Select your existing managed identity, SSH, or Tower Agent credentials, or select **+** and **SSH** or **Tower Agent** to add new credentials.

> 1.  Enter the absolute path of the **Work directory** to be used on the cluster. The `TW_AGENT_WORKDIR` and `TW_AGENT_USER` variables can be used when entering the file system path.

Reviewer: ```suggestion
1.  Enter the absolute path of the **Work directory** to be used on the cluster. You can use the `TW_AGENT_WORKDIR` and `TW_AGENT_USER` variables in the file system path.
```
